psl-parser/src/main/java/org/linqs/psl/parser/CommandLineLoader.java

```diff
@@ -87,22 +79,25 @@
     public static final String OPTION_VERSION_LONG = "version";
 
     public static final String DEFAULT_H2_DB_PATH =
-    Paths.get(System.getProperty("java.io.tmpdir"),
-    "cli_" + System.getProperty("user.name") + "@" + getHostname()).toString();
+                    Paths.get(System.getProperty("java.io.tmpdir"),
+                    "cli_" + System.getProperty("user.name") + "@" + getHostname()).toString();
     public static final String DEFAULT_POSTGRES_DB_NAME = "psl_cli";
-    public static final String DEFAULT_IA = MPEInference.class.getName();
-    public static final String DEFAULT_WLA = MaxLikelihoodMPE.class.getName();
-
+    private static final String DEFAULT_IA = MPEInference.class.getName();
+    private static final String DEFAULT_WLA = MaxLikelihoodMPE.class.getName();
+    
+    public static Options unparsedOptions;
```

Comment:
What's with the name here?
This makes it sound like arguments on the command line that did not match any specified options.

Reply:
Can you please suggest a name?